tests/conftest.py

```diff
@@ -0,0 +1,255 @@
+import pytest
+import json
+import logging
+import os
+import time
+import yaml
+
+from oc.openshift_ops import OCP
+from ocs.exceptions import CommandFailed, CephHealthException
+from ocs.utils import create_oc_resource, apply_oc_resource
+from ocsci import config
+from ocsci.config import RUN, ENV_DATA, DEPLOYMENT
+from utility import templating
+from utility.aws import AWS
+from utility.retry import retry
+from utility.utils import destroy_cluster, run_cmd, get_openshift_installer, get_openshift_client, is_cluster_running
```

Comment:
I know the rule isn't set in our flake8 check yet but let's keep this line < 79 chars.

Reply:
Will do